openmetadata-ui/src/main/resources/ui/playwright/utils/alert.ts

```diff
@@ -763,7 +763,6 @@ export const saveAlertAndVerifyResponse = async (page: Page) => {
 
     expect(response.status()).toEqual(201);
   });
-  await toastNotification(page, 'Alerts created successfully.');
```

Comment:
same

Reply:
I think while saving, I mistakenly removed it and missed it.